app/views/contribute/index.erb

```diff
@@ -115,15 +115,15 @@
           </p>
 
           <p>
-            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#beta-testing-a-language-track">beta testing</a> in the Contributing Guide.
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#beta-testing-a-language-track">beta testing</a> in the Contributing Guide.
           </p>
 
           <p>
             Didn't find a language? We're always looking to increase our supported language tracks. <a href="mailto:kytrinyx@exercism.io">Email Katrina</a>, she'll be glad to help.
           </p>
 
            <p>
-            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#starting-a-new-track">starting a new track</a> in the Contributing Guide.
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#starting-a-new-track">starting a new track</a> in the Contributing Guide.
```

Comment:
Let's link to this instead: https://github.com/exercism/request-new-language-track/issues/new